scripts/clean-db-dump.sql

```diff
@@ -112,6 +112,47 @@ WHERE declaration IS NOT NULL
   AND declaration::varchar != 'null'
   AND declaration::varchar != '{}';
 
+-- Create some draft, open and closed briefs
+UPDATE
+  briefs
+SET
+  published_at =
+  CASE
+    WHEN MOD(id, 5) = 0 THEN NULL
+    WHEN MOD(id, 4) = 0 THEN now() - '1 day'::INTERVAL
+    ELSE now() - '2 weeks 1 day'::INTERVAL
+  END;
+
+-- Create some fake draft and closed briefs using eligible suppliers
+INSERT INTO brief_responses (
+      data,
+      brief_id,
+      supplier_id,
+      created_at,
+      submitted_at
+)
+SELECT
+    '{"essentialRequirements": [{"evidence": "They have the correct number of hooves."}, {"evidence": "So shiny..."}, {"evidence": "All at least 50"}], "niceToHaveRequirements": [{"yesNo": false}, {"yesNo": true, "evidence": "No jump is too high."}, {"yesNo": true, "evidence": "NEIGH"}], "availability": "09/09/17", "essentialRequirementsMet": true, "respondToEmailAddress": "example-email@gov.uk"}'::json,
+    eligible_brief_supplier_pairings.brief_id,
+    eligible_brief_supplier_pairings.supplier_id,
+    CASE
+      WHEN MOD(id, 5) = 0 THEN NULL
+      ELSE now()
+    END,
+    now()
```

Comment:
This might be my rusty SQL knowledge showing, but does this always put `now()` into `submitted_at` and the case statement into `created_at`? Shouldn't it be the other way around?